classes/persunclass.rb

```diff
@@ -0,0 +1,17 @@
+# suberclass
```

Comment:
- [ ] To follow the class and file naming convention, kindly change the name of the file of this class from `persunclass.rb` to `person.rb`. This will also ensure correct import of the the person.rb file in